skrub/tests/test_similarity_encoder.py

```diff
@@ -322,10 +339,14 @@ def test_check_fitted_super_vectorizer() -> None:
     sim_enc.transform(X)
 
 
-def test_inverse_transform():
+@pytest.mark.parametrize("px", MODULES)
+def test_inverse_transform(px):
     encoder = SimilarityEncoder()
-    encoder.set_output(transform="pandas")
-    X = pd.DataFrame({"A": ["aaa", "aax", "xxx"], "B": ["bbb", "bby", "yyy"]})
+    if px is pd:
+        encoder.set_output(transform="pandas")
+    elif px is pl:
+        encoder.set_output(transform="polars")
```

Comment:
The issue is that this is not possible ([yet](https://github.com/scikit-learn/scikit-learn/pull/27315)!). I guess we can only `xfail` that one.